### ISSUE TITLE
Problems with config.layer_transforms and camera movements

### ISSUE THREAD

**Issue by kigyo** [renpy/renpy#5352]
Another problem I ran into using 8.3.0 was when using the new `config.layer_transforms`, which I am very excited about!
I thought it would save me the effort of applying my `CONTRAST()` transform on all screens individually, but unfortunately it does not mesh well with camera movements:

https://github.com/renpy/renpy/assets/76883792/17263fda-3fd4-4c64-939b-4fc75031328e

As you can see, each end of a say statement, each {w} statement, as well as each hover event on the quick menu, restarts the previous camera animation.

Here is the code for my contrast transform:
```
transform image_edit(saturation=1.0, contrast=1.0):
    matrixcolor SaturationMatrix(saturation) * ContrastMatrix(contrast)

def contrast_changer(child, **properties):
        if True:
            return At(child, image_edit(persistent.saturation, persistent.contrast))
        else:
            return Fixed(child, fit_first=True, **properties)

    CONTRAST = renpy.curry(contrast_changer)

define config.layer_transforms[None] = [ CONTRAST() ]
```

And here is my camera function:
```
transform camera_movement(warper, t, xpos, ypos, zoom, rotate):
    perspective True subpixel True
    parallel:
        function moving_camera
    parallel:
        warp warper t xpos xpos ypos ypos zoom zoom rotate rotate xanchor 0.5 yanchor 0.5
        
python:
    def cam(warp=linear, t=1, x=None, y=None, z=None, rotate=0):
        if persistent.camera_move:
            if renpy.is_skipping():
                t = 0
            if x and y and z:
                renpy.show_layer_at([camera_movement(warp, t, x, y, z, rotate)], camera=True)
            elif rotate:
                renpy.show_layer_at([camera_movement_rot(warp, t, rotate)], camera=True)
            if not persistent.camera_move:
                renpy.with_statement(dis)
```

**Comment by renpytom** [comment]
Please don't put pseudo-tags in your issues, just state the version it applies to in the text.

**Comment by renpytom** [comment]
Fixed by 609930bedba3b72ec597b533a73d4debea31cffa, which is in the nightly fix branch.

**Comment by kigyo** [comment]
> Fixed by [609930b](https://github.com/renpy/renpy/commit/609930bedba3b72ec597b533a73d4debea31cffa), which is in the nightly fix branch.

Can confirm, this fixes it. Thank you!
